File: AreaOfRectangle.java
/*
=====================================================
PRACTICAL: Area of Rectangle in Java
=====================================================

AIM:
- To write a Java program that calculates the area of a rectangle.

OBJECTIVE:
- To read length and breadth from user.
- To apply formula: Area = length × breadth.
- To display calculated area.

TOOLS:
- JDK 8+ (javac, java)
- Any IDE or text editor

ALGORITHM:
1) Import Scanner class.
2) Read length and breadth from user.
3) Compute area = length * breadth.
4) Print calculated area.
5) End program.

STEPS TO RUN:
- Save as: AreaOfRectangle.java
- Compile: javac AreaOfRectangle.java
- Run:     java AreaOfRectangle

=====================================================
*/

import java.util.Scanner;

public class AreaOfRectangle {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter length of rectangle: ");
        double length = sc.nextDouble();

        System.out.print("Enter breadth of rectangle: ");
        double breadth = sc.nextDouble();

        double area = length * breadth;

        System.out.println("Area of Rectangle = " + area);

        sc.close();
    }
}

/*
================= SAMPLE OUTPUT =================

Enter length of rectangle: 10
Enter breadth of rectangle: 5
Area of Rectangle = 50.0

Enter length of rectangle: 7.5
Enter breadth of rectangle: 4
Area of Rectangle = 30.0

=================================================

RESULT:
- Successfully implemented program to calculate area of rectangle in Java.
- Program applies formula Area = length × breadth.
=================================================
*/

File: PHP/areaofrectangle.php
<?php
/*
=====================================================
PRACTICAL: PHP Program to Find Area of Rectangle
=====================================================

AIM:
- To write a PHP program that calculates
  the area of a rectangle.

OBJECTIVE:
- To read length and breadth from user.
- To apply formula: Area = length × breadth
- To display the result.

TOOLS:
- PHP 7+/8+
- Any Web Server (XAMPP/WAMP/LAMP)

ALGORITHM:
1) Accept length and breadth from user.
2) Apply formula: Area = length × breadth
3) Display calculated area.
4) End.

STEPS TO RUN:
- Save file as: area_rectangle.php
- Open in browser: http://localhost/area_rectangle.php
- Enter values and check.

=====================================================
PHP CODE (area_rectangle.php)
=====================================================
*/
?>

<!DOCTYPE html>
<html>
<head>
  <meta charset="UTF-8">
  <title>Area of Rectangle</title>
</head>
<body>
  <h1>Area of Rectangle (PHP)</h1>

  <form method="post">
    <label>Enter length:</label><br>
    <input type="number" name="length" step="0.01" min="0" required><br><br>

    <label>Enter breadth:</label><br>
    <input type="number" name="breadth" step="0.01" min="0" required><br><br>

    <button type="submit">Calculate</button>
  </form>

<?php
if ($_SERVER["REQUEST_METHOD"] === "POST") {
    $length  = (float)$_POST["length"];
    $breadth = (float)$_POST["breadth"];

    $area = $length * $breadth;

    echo "<p>Length: $length , Breadth: $breadth</p>";
    echo "<h3>Area of Rectangle = " . round($area, 2) . "</h3>";
}
?>

<?php
/*
================= SAMPLE OUTPUT =================

Input: Length = 10, Breadth = 5  
Output: Area of Rectangle = 50  

Input: Length = 7.5, Breadth = 3.2  
Output: Area of Rectangle = 24  

=================================================
RESULT:
- Successfully implemented a PHP program that
  calculates area of a rectangle.
=================================================
*/
?>
</body>
</html>
